nessclient/event.py

```diff
@@ -146,6 +146,7 @@ class RequestID(Enum):
         VIEW_STATE = 0x16
         PANEL_VERSION = 0x17
         AUXILIARY_OUTPUTS = 0x18
+        UNKNOWN = 0x30
```

Comment:
I think it might be clearer to define it more specifically as being an undocumented value? WDYT?

```suggestion
        UNDOCUMENTED_01 = 0x30
```